pyproject.toml

```diff
@@ -9,7 +9,7 @@ description = ""
 authors = []
 
 [tool.poetry.dependencies]
-python = "^3.8.1"  # ^3.8.1 required by flake8
+python = "^3.8.6"
```

Comment:
This is the minimum Python version required by juju `3.6.0.0`. Otherwise, installation fails.

Reply:
Done in commit https://github.com/canonical/mysql-router-operator/pull/197/commits/5bee0c103d13ec5453a0ed1106000a3c59d59553.